plugins/inputs/t128_graphql/query_builder.go

```diff
@@ -18,7 +18,7 @@ const (
 BuildQuery first creates an intermediary query object that is traversed by buildQueryBody() in pre-order
 
 Args:
-	entryPoint example - "allRouters[name:ComboEast]/nodes/nodes[name:combo-east]/nodes/arp/nodes"
+	entryPoint example - "allRouters(name:\"ComboEast\)/nodes/nodes(name:\"combo-east\")/nodes/arp/nodes"
```

Comment:
Would the user have to add all this `\"` escaping when configuring the query too?